sdk/rust/oak/src/io/sender.rs

```diff
@@ -43,12 +43,14 @@ impl<T: Encodable> SenderExt<T> for Sender<T> {
     fn send(&self, t: &T) -> Result<(), OakError> {
         let message = t.encode()?;
         crate::channel_write(&self.handle, &message.bytes, &message.handles)?;
+        drop_handles(message);
```

Comment:
I am not sure why the handles need to be dropped. Are not they borrowed or moved?

Reply:
During encoding we extract the raw handle values from the message, and replace them with `INVALID_HANDLE`: https://github.com/project-oak/oak/blob/main/oak_io/src/handle.rs#L202.

When this message is dropped, all handle values are invalid, so no handles are dropped. Instead we need to manually drop the raw handle values here. I hope that makes sense?

Reply:
Yes. Thanks. I had overlooked a `clone` :)